build.gradle

```diff
@@ -78,7 +78,8 @@ dependencies {
   shadow localGroovy()
   shadow gradleApi()
 
-  implementation files ("nexus-platform-api/build/libs/nexus-platform-api-${version}.jar")
+  //implementation files ("nexus-platform-api/build/libs/nexus-platform-api-${version}.jar")
```

Comment:
I believe this is needed just locally to workaround IDE issues.

Reply:
> I believe this is needed just locally to workaround IDE issues.

:sweat_smile: beab896d